### PR TITLE
SDK-487 Build SDK task for codeception execution

### PR DESCRIPTION
**Pull request by ScarIetSky**
SDK-487 Build SDK task for codeception execution 

* Added a task for the execution of the acceptance tests

### CODE REVIEW

File: tests/Sdk/Acceptance/Extension/Tasks/ESLintTaskCest.php [hunk 1]
```diff
@@ -53,7 +53,6 @@ public function testFilesDontContainViolations(AcceptanceTester $I): void
         Assert::assertFileExists(
             $I->getPathFromProjectRoot('reports/' . static::COMMAND . '.violations.yaml', static::PROJECT_DIR),
         );
-        Assert::assertStringNotContainsString('Violations found in files', $process->getOutput());
```
Comment (miroshnikov-spryker) [inline tests/Sdk/Acceptance/Extension/Tasks/ESLintTaskCest.php:56]: Why did you delete these assetions?

File: src/Extension/Tasks/AcceptanceTestingTask.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+id: 'tests:sdk:acceptance'
+short_description: 'Run acceptance tests'
+help: ~
+stage: build
+version: 0.1.0
+command: 'php %sdk_dir%/vendor/bin/codecept run acceptance'
```
Comment (miroshnikov-spryker) [inline src/Extension/Tasks/AcceptanceTestingTask.yaml:7]: Codeception command produces an error report as a result of it's work therefore it needs to be parsed and converted into SDK's Violation Report. I assume codecept command has an option to change the report format to json so that it can be parsed and converted more easy.

Comment (ScarIetSky) [reply src/Extension/Tasks/AcceptanceTestingTask.yaml]: I added creation of the report after executing the command

**Review by miroshnikov-spryker — APPROVED** [review]
